ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/PreviousOrganisation.java

```diff
@@ -22,27 +22,30 @@ public class PreviousOrganisation {
   @JsonProperty("FromTimeStamp")
   @JsonSerialize(using = LocalDateTimeSerializer.class)
   @JsonDeserialize(using = LocalDateTimeDeserializer.class)
-  @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS")
+  @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS",
+          with = JsonFormat.Feature.ACCEPT_CASE_INSENSITIVE_PROPERTIES
```

Comment:
does this cope with the 6s I think you mentioned experiencing too or is that no longer a problem?

Reply:
I think that was happening with `requestTimestamp` in changeOrganisationPolicy - we noticed that no timestamp pattern was specified for that field in [AAC API](https://github.com/hmcts/aac-manage-case-assignment/blob/a80abcea28cbd567ab5958d0ca24b7b1e33387a6/src/main/java/uk/gov/hmcts/reform/managecase/domain/ChangeOrganisationRequest.java#L36) and removing the pattern from our code stopped the issue. It might make sense to remove the pattern from these fields too? Although if it's working fine during testing then I don't have a strong opinion on that